package.json

```diff
@@ -3,8 +3,10 @@
   "version": "0.1.1",
   "private": true,
   "dependencies": {
+    "leaflet": "^1.8.0",
```

Comment:
What is this leaflet dependency? What does it do?

Reply:
It is an opensource JavaScript library for interactive maps.

I am [working on a library](https://github.com/clairBuoyant/worldwind-react) that may replace this leaflet dependency for us in the future. ([ref](https://github.com/clairBuoyant/web/issues/3#issuecomment-1183431593))

Reply:
For more information regarding the new dependencies, these links would be a great starting point:
- [Leaflet documentation](https://leafletjs.com/)
- [React Leaflet documentation](https://react-leaflet.js.org/docs/start-introduction/)

The latter dependency integrates leaflet with React.